pom.xml

```diff
@@ -895,6 +895,41 @@
 					<header>${project.basedir}/license-header.txt</header>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>com.github.spotbugs</groupId>
+				<artifactId>spotbugs-maven-plugin</artifactId>
+				<version>3.1.12</version>
```

Comment:
could this be the latest version of  spotbugs-maven-plugin if not what really happens when you try to use the latest version?

Reply:
@HerbertYiga Updated to the latest version of  `spotbugs-maven-plugin `